Domain/Validators/ProductValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+using LearnNet_CatalogService.Data.Entities;
+
+namespace LearnNet_CatalogService.Domain.Validators
+{
+    public class ProductValidator : AbstractValidator<Product>
```

Comment:
instead of creating Validator classes for entities, restrict allowed values in OnModelCreating method

Reply:
OnModelCreating isn't as flexible as it needs to be, but partly fixed